cli-output/src/cli_output.rs

```diff
@@ -2147,7 +2153,12 @@ pub fn return_signers_data(tx: &Transaction, config: &ReturnSignersConfig) -> Cl
                 bad_sig.push(key.to_string());
             }
         });
-    let message = if config.dump_transaction_message {
+    let dumped_transaction = if config.dump_transaction_message {
+        Some(base64::encode(serialize(tx).unwrap()))
```

Comment:
i think we need special handling for the transaction case when it is only partially signed. which probably means a discrete cli flag

Reply:
hmm, could this be a blocker for merging this pr? if possible, i want to avoid all the way down plumbing entailing adding a discrete cli flag...

as said at this pr's description, i don't think it's unacceptably bad to dump partially signed transactions as part of default dump behavior.

Reply:
@t-nelson friendly ping for the above proposal. :)

Reply:
> I have to admit that --output json still dumps the new "transaction": ... field unconditionally still. ;)

That's fine.  IMO the JSON output should always have all fields, the optional stuff is for interactive text interface

Reply:
> That's fine. IMO the JSON output should always have all fields, the optional stuff is for interactive text interface

+1